### ISSUE TITLE
Request to reset scanned locations

### ISSUE THREAD

**Issue by kannienchenman** [mchristopher/PokemonGo-DesktopMap#266]
Please add an option to whipe all the "scanned location data".
It is sometimes a chaos of all kinds of different color circles, because they all overlap each other.

**Comment by AVellucci** [comment]
The circles should not be chaotically overlapping each other.
Can you provide a screenshot for this issue?
 As well as steps taken to reproduce this.

**Comment by Draag** [comment]
When moving the scan location, the previous "Scanned Locations" will remain and cycle to red until they expire after 15 minutes(?) and ultimately disappear.  Keeping the scan on the same location only results in 2 shades of green without 'chaotic overlap'. 

I believe the user is requesting a feature to clear all 'Scanned Locations' indicators. I have found that this bogs down slower systems when multiple 'Scanned Locations' fields are cycling down.

To reproduce, complete a [distance] 4 scan, move the location selector about 1 circle away, scan again, then down 1 and scan again. It will eventually become a mess of reds, yellows and greens while the expiring 'Scanned Locations' fade.

**Comment by RyusuiJL** [comment]
I would like to add to this request.  I like to keep my scan radius low (2 or 3) so that I can move it around as need be.  But as they overlap a lot, it gets hard to follow the areas I have and have not scanned.

In addition to the original request, I would also like add the function to have the scanned locations reset anytime you move the marker.  This could maybe be a toggle.
